notebooks.go

```diff
@@ -139,6 +139,11 @@ type notebooksresponse struct {
 	Notebooks []notebookresponse `json:"notebooks"`
 }
 
+type notebookapiresponse struct {
```

Comment:
Is there a reason you need that specific type of notebook response? Can't you just use the "getnotebookresponse" that is just underneath, and did something like the getNotebook function that we already have https://github.com/StatCan/jupyter-apis/blob/63ceeaca2ff2f85faf2a962acb90fd0a7844c075/notebooks.go#L1065?

Reply:
cause i want the normal data and the api response. and the other one has a kybeflow format, this one has our final

Reply:
makes sense